bin/merge_and_align_libraries_from_easypqp.py

```diff
@@ -51,7 +51,7 @@ def align_libs(reference, other, rsq_threshold):
 
     # calibrate rt alignment
     df_rt_merged = pd.merge(df_rt, df_rt_II, on='Key')
-    slope, intercept, r_value, p_value, std_err = stats.linregress(df_rt_merged['NormalizedRetentionTime_x'], df_rt_merged['NormalizedRetentionTime_y'])
+    slope, intercept, r_value, p_value, std_err = stats.linregress(df_rt_merged['NormalizedRetentionTime_y'], df_rt_merged['NormalizedRetentionTime_x'])
```

Comment:
maybe add a comment what y and x are here

Reply:
done